x-pack/metricbeat/module/oracle/tablespace/mocks_test.go

```diff
@@ -0,0 +1,92 @@
+		{FileName: sql.NullString{String: "/u02/app/oracle/oradata/ORCLCDB/orclpdb1/sysaux03.dbf", Valid: true}, FileID: sql.NullInt64{Int64: 182, Valid: true}, TablespaceName: sql.NullString{String: "SYSAUX", Valid: true}, Status: sql.NullString{String: "AVAILABLE", Valid: true}, MaxFileSizeBytes: sql.NullInt64{Valid: true, Int64: 9999996}, AvailableForUserBytes: sql.NullInt64{Int64: 99999996, Valid: true}, OnlineStatus: sql.NullString{String: "ONLINE", Valid: true}, FileSizeBytes: sql.NullInt64{Valid: true, Int64: 9999992}},
+
+		{FileName: sql.NullString{String: "/u02/app/oracle/oradata/ORCLCDB/orclpdb1/system01.dbf", Valid: true}, FileID: sql.NullInt64{Int64: 18, Valid: true}, TablespaceName: sql.NullString{String: "SYSTEM", Valid: true}, Status: sql.NullString{String: "AVAILABLE", Valid: true}, MaxFileSizeBytes: sql.NullInt64{Valid: true, Int64: 9999994}, AvailableForUserBytes: sql.NullInt64{Int64: 9999994, Valid: true}, OnlineStatus: sql.NullString{String: "ONLINE", Valid: true}, FileSizeBytes: sql.NullInt64{Valid: true, Int64: 999990}},
+		{FileName: sql.NullString{String: "/u02/app/oracle/oradata/ORCLCDB/orclpdb1/temp012017-03-02_07-54-38-075-AM.dbf", Valid: true}, FileID: sql.NullInt64{Int64: 18, Valid: true}, TablespaceName: sql.NullString{String: "TEMP", Valid: true}, Status: sql.NullString{String: "AVAILABLE", Valid: true}, MaxFileSizeBytes: sql.NullInt64{Valid: true, Int64: 9999994}, AvailableForUserBytes: sql.NullInt64{Int64: 9999994, Valid: true}, OnlineStatus: sql.NullString{String: "ONLINE", Valid: true}, FileSizeBytes: sql.NullInt64{Valid: true, Int64: 999991}},
+		{FileName: sql.NullString{String: "/u02/app/oracle/oradata/ORCLCDB/orclpdb1/undotbs01.dbf", Valid: true}, FileID: sql.NullInt64{Int64: 18, Valid: true}, TablespaceName: sql.NullString{String: "UNDOTBS1", Valid: true}, Status: sql.NullString{String: "AVAILABLE", Valid: true}, MaxFileSizeBytes: sql.NullInt64{Valid: true, Int64: 9999994}, AvailableForUserBytes: sql.NullInt64{Int64: 9999994, Valid: true}, OnlineStatus: sql.NullString{String: "ONLINE", Valid: true}, FileSizeBytes: sql.NullInt64{Valid: true, Int64: 999992}},
+		{FileName: sql.NullString{String: "/u02/app/oracle/oradata/ORCLCDB/orclpdb1/users01.dbf", Valid: true}, FileID: sql.NullInt64{Int64: 18, Valid: true}, TablespaceName: sql.NullString{String: "USERS", Valid: true}, Status: sql.NullString{String: "AVAILABLE", Valid: true}, MaxFileSizeBytes: sql.NullInt64{Valid: true, Int64: 9999994}, AvailableForUserBytes: sql.NullInt64{Int64: 9999994, Valid: true}, OnlineStatus: sql.NullString{String: "ONLINE", Valid: true}, FileSizeBytes: sql.NullInt64{Valid: true, Int64: 999993}},
```

Comment:
Nit. Newlines are cheap :slightly_smiling_face:, unwrap these for readability?

Reply:
I prefer to leave them like this because file is more readable (data on those structs no, but they are mocks so...)

Reply:
Ok